rotkehlchen/data/importer.py

```diff
@@ -858,10 +862,12 @@ def _consume_nexo(self, csv_row: Dict[str, Any]) -> None:
                 asset=asset,
                 rate=None,
                 rate_asset=None,
-                link=None,
+                link=transaction,
                 notes=f'{entry_type} from Nexo',
             )
             self.db_ledger.add_ledger_action(action)
+        elif entry_type in ignored_entries:
```

Comment:
I think this approach is not really future-safe. Instead what you should go is ignore any unknown/unexpected entries with a warning.

And iff all the expected entries were not there raise an exception.

Reply:
At the moment I rise a warning if a unknow row is detected. In this case there are two rows that include information for internal operations of nexo and contain information that we get from other rows